config.go

```diff
@@ -168,10 +168,15 @@ const (
 	// TODO(halseth): find a more scientific choice of value.
 	defaultMaxLocalCSVDelay = 10000
 
-	// defaultChannelCommitInterval is the default maximum time between receiving a
-	// channel state update and signing a new commitment.
+	// defaultChannelCommitInterval is the default maximum time between
+	// receiving a channel state update and signing a new commitment.
```

Comment:
More curiosity here! Has anyone ever played around with these batch parameters? Could be cool to modify one of the payment benchmark tests to sweep these to get an idea.

Reply:
Not to my knowledge. The bench tests would vary based on the OS, db backend, chain backend and its config. The itests in lnd has covered 7 cases based on the combinations.